### ISSUE TITLE
Codepage conversion error, from Unicode (codepage 4103) to ISO 8859-1 (codepage 1100)

### ISSUE THREAD

**Issue by jaysonwcs** [abapinho/abapTimeMachine#127]
Tried to use ZBLAME to analyse a program and got this error.

[Dump.pdf](https://github.com/abapinho/abapTimeMachine/files/10580542/Dump.pdf)

**Comment by nununo** [comment]
I was able to recreate the dump. Unfortunately I was still unable to fix it. I mean, I can make the dump go away but then all accents become a big mess. I'll keep trying but any suggestion is welcome.

**Comment by jaysonwcs** [comment]
I tried installing again on the same system and, for some reason, it worked flawlessly.
I know the bug wasn't officially fixed, but it worked.

The other try was made on February, so I may not remember correctly what I did, but I believe I tried to install it with the system logged in portuguese instead of english. I tried to install it today with the system logged in english (and the package created with english as main language) and it worked. Maybe the language was the problem? I'm not sure.
